app/models/mixins/cockpit_support_mixin.rb

```diff
@@ -0,0 +1,33 @@
+module CockpitSupportMixin
+  extend ActiveSupport::Concern
+
+  included do
+    supports :cockpit_console do
+      message = _('The web-based console is not available because the')
+      if respond_to?(:platform) && platform == 'windows'
+        unsupported_reason_add(:cockpit_console, _("#{message} Windows platform is not supported"))
```

Comment:
Will it try to double translate `message`?

Reply:
@bdunne - good question. Guess the first one shouldn't be translated.